core/micrometer/build.gradle.kts

```diff
@@ -21,12 +21,32 @@ val jupiterVersion: String by project
 val okHttpVersion: String by project
 val jodahFailsafeVersion: String by project
 val micrometerVersion: String by project
+val rsApi: String by project
 
 
 dependencies {
     api(project(":core:base"))
     api("com.squareup.okhttp3:okhttp:${okHttpVersion}")
     api("io.micrometer:micrometer-core:${micrometerVersion}")
+
+    testImplementation(project(":extensions:api:observability"))
+    testImplementation(testFixtures(project(":extensions:azure:azure-test")))
+    testImplementation(testFixtures(project(":launchers:junit")))
+    testImplementation(testFixtures(project(":common:util")))
+
+    runtimeOnly(project(":extensions:http:jersey-micrometer"))
+    runtimeOnly(project(":extensions:http:jetty-micrometer"))
```

Comment:
Can it be `testRuntimeOnly` or even better `testCompileOnly` as suggested in [upstream PR 911.](https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/pull/911)